frontend/src/components/Shared/Navbar/Navbar.jsx

```
@@ -53,163 +49,141 @@ const NavigationBar = () => {
-                        src={user.photoURL || "https://flowbite.com/docs/images/people/profile-picture-5.jpg"}
-                        alt="User profile"
-                        className="h-full w-full object-cover"
-                      />
-                    </div>
-                  </button>
+        {/* User Profile Dropdown */}
+        <div className="relative" ref={userMenuRef}>
+          <button 
+            onClick={() => setShowUserMenu(!showUserMenu)}
+            className="flex items-center focus:outline-none"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)